src/improvements/template/TransferOwners.sol

```diff
@@ -138,6 +142,28 @@ contract TransferOwners is L2TaskBase {
         IOwnable(_target).transferOwnership(_newOwner);
     }
 
+    /// @notice Performs an ownership transfer by writing the new owner directly to the owner slot.
+    function performWethOwnershipTransfer(address _target, address _newOwner) internal {
+        _writeToProxy(_target, bytes32(uint256(51)), bytes32(uint256(uint160(_newOwner))));
```

Comment:
Slot 0 is not the owner on the latest DelayedWETH , we should have a semver check for safety (maybe checking for the `owner()` method could work as well)

Reply:
Nevermind, I see this is wrapped in a `_isDWETHOwnable` check

Reply:
I could move that check within this function to ensure they are tightly coupled, if so then I think the name would need to be improved, maybe `performTransferIfOwnable`?

Reply:
ef841a75b947